src/render-line-item.js

```diff
@@ -0,0 +1,31 @@
+import { getLineTotal } from './register.js';
+import { toUSD } from '../src/format.js';
+
+function renderLineItem(lineItem, gear){
+
+    const tr = document.createElement('tr');
+
+    const classCell = document.createElement('td');
```

Comment:
"classCell" is a little confusing as a name here - maybe "quantityCell" would work?